File: poo/2. User Registration at a Gym/cliente.php
<?php
    class Cliente extends Person {
        private ?string $name;
        private ?string $lastName;
        private float $weight;
        private float $stature;

        public function __construct(?string $name, ?string $lastName, float $weight, float $stature) {
            parent::__construct(1080292891);
            $this->name = $name;
            $this->lastName = $lastName;
            $this->weight = $weight;
            $this->stature = $stature;
        }

        public function getName(): ?string {
            return $this->name;
        }
        public function getLastName(): ?string {
            return $this->lastName;
        }
        public function getWeight(): float {
            return $this->weight;
        }
        public function getStature(): float {
            return $this->stature;
        }
    }
?>

File: poo/2. User Registration at a Gym JS/cliente.java
public class Cliente extends Person {
    private String name;
    private String lastName;
    private double weight;
    private double stature;

    public Cliente(String name, String lastName, double weight, double stature) {
        super(1080292891); // ID de la persona
        this.name = name;
        this.lastName = lastName;
        this.weight = weight;
        this.stature = stature;
    }

    public String getName() {
        return name;
    }

    public String getLastName() {
        return lastName;
    }

    public double getWeight() {
        return weight;
    }

    public double getStature() {
        return stature;
    }
}
